meltdown-mitigation/conditionals.py

```diff
@@ -37,20 +34,32 @@ def reactor_efficiency(voltage, current, theoretical_max_power):
     where generated power = voltage * current
     """
 
-    pass
+    efficiency: Union[int, float] = voltage * current / theoretical_max_power * 100
 
+    if efficiency < 30:
+        return "black"
+    elif efficiency < 60:
+        return "red"
+    elif efficiency < 80:
+        return "orange"
+    return "green"
 
-def fail_safe(temperature, neutrons_produced_per_second, threshold):
-    """Assess and return status code for the reactor.
 
-    :param temperature: value of the temperature in kelvin (integer or float)
-    :param neutrons_produced_per_second: neutron flux (integer or float)
-    :param threshold: threshold (integer or float)
-    :return: str one of: 'LOW', 'NORMAL', 'DANGER'
+def fail_safe(temperature, neutrons_produced_per_second, threshold: Union[int, float]) -> str:
```

Comment:
Check tests file. Although it can be float potentially, the data shows that `temperature`, `neutrons_produced_per_second`, and `threshold` are all `int`s. Doesn't matter much though.

Reply:
Yep, they are all `int`s,  but I'll leave it as it is for now, because their types are declared in docstring above.

Reply:
Fair enough